wordpress-mobile/AztecEditor-iOS
# Issue/994 crash gb missing closing element

Fixes #994 

This PR adds a fix for the crash when a block if missing a closing comment.
Maybe it's not the perfect fix, but it's small and doesn't have a big impact on the rest of the code.
I tried to do fix on the input but the code was getting more complicated than expected. What do you think @diegoreymendez ?

To test:
 - Start the Gutenberg demo
 - Switch to HTML mode
 - Copy paste the following code

```

<ul>
    <li>Media library/HTML for images, multimedia and approved files.</li>
    <li>Pasted links for embeds.</li>
    <li>Shortcodes for specialized assets from plugins.</li>
    <li>Featured images for the image at the top of a post or page.</li>
    <li>Excerpts for subheads.</li>
    <li>Widgets for content on the side of a page.</li>
</ul>
```

Switch to visual and then to HTML again and check that app not crashes
